tests/test_cli.py

```diff
@@ -92,55 +88,6 @@ def test_lack_of_calling_json_deserializer_when_no_query_appears(
             cli_download.main()
         mock_json.assert_not_called()
 
-    @mock.patch('geospaas_processing.downloaders.DownloadManager.__init__', return_value=None)
-    @mock.patch('geospaas_processing.downloaders.DownloadManager.download')
-    def test_correct_call_DownloadManager_without_file_prefix(
-            self, mock_download_method, mock_download_manager_init):
-        """shall return the proper call for the case of lack of file prefix ('-p') in arguments"""
-        sys.argv.remove('-p')
-        sys.argv.remove('-r')
-        sys.argv[4] = '2019-10-22'
-        arg = cli_download.cli_parse_args()
-        cli_download.main()
-        self.assertIn({
-            'download_directory': '/test_folder/%Y_nh_polstere',
-            'geographic_location__geometry__intersects':
-            GEOSGeometry('POLYGON ((-22 84, -22 74, 32 74, 32 84, -22 84))'),
-            'max_downloads': 100,
-            'provider_settings_path': '/config_folder/config_file.yml',
-            'time_coverage_end__lte': datetime(2020, 8, 22, 0, 0, tzinfo=tzutc()),
-            'time_coverage_start__gte': datetime(2019, 10, 22, 0, 0, tzinfo=tzutc()),
-            'dataseturi__uri__contains': 'osisaf',
-            'source__instrument__short_name__icontains': 'AMSR2',
-            'use_file_prefix': False,
-            'save_path': True
-        }, mock_download_manager_init.call_args)
-
-    @mock.patch('geospaas_processing.downloaders.DownloadManager.__init__', return_value=None)
-    @mock.patch('geospaas_processing.downloaders.DownloadManager.download')
-    def test_correct_call_DownloadManager_with_file_prefix(
-            self, mock_download_method, mock_download_manager_init):
-        """
-        shall return the proper call for the case of lack of two definite time points in arguments
-        """
-        sys.argv.remove('-r')
-        sys.argv[4] = '2019-10-22'
-        arg = cli_download.cli_parse_args()
-        cli_download.main()
-        self.assertIn({
-            'download_directory': '/test_folder/%Y_nh_polstere',
-            'geographic_location__geometry__intersects':
-            GEOSGeometry('POLYGON ((-22 84, -22 74, 32 74, 32 84, -22 84))'),
-            'max_downloads': 100,
-            'provider_settings_path': '/config_folder/config_file.yml',
-            'time_coverage_end__lte': datetime(2020, 8, 22, 0, 0, tzinfo=tzutc()),
-            'time_coverage_start__gte': datetime(2019, 10, 22, 0, 0, tzinfo=tzutc()),
-            'dataseturi__uri__contains': 'osisaf',
-            'source__instrument__short_name__icontains': 'AMSR2',
-            'use_file_prefix': True,
-            'save_path': True
-        }, mock_download_manager_init.call_args)
```

Comment:
this test is the most comprehensive one compared to the one next to it and compared to the previous one. So keep this one and delete the next and the previous one entirely. Because this one has the geometry in it and by deleting that we do not test the geometry input.

Reply:
Isn't the geometry input tested in `test_correct_call_DownloadManager_with_relative_time()`?

Reply:
yes.no need to change